dffml/base.py

```diff
@@ -275,6 +283,8 @@ def args(cls, args, *above) -> Dict[str, Arg]:
             # HACK For detecting dataclasses._MISSING_TYPE
             if "dataclasses._MISSING_TYPE" not in repr(field.default):
                 arg["default"] = field.default
+            if "dataclasses._MISSING_TYPE" not in repr(field.default_factory):
+                arg["default_factory"] = field.default_factory
```

Comment:
Since `args()` often get's `json.dump`'d (and won't output the `lambda` correctly), we should probably do `arg["default"] = field.default_factory()`
and then in `config_get` remove the `"default_factory"` logic, and just always do `if "default" in arg: return copy.deepcopy(arg["default"])`